convex_hull_triangulation.py

```diff
@@ -0,0 +1,73 @@
+from geometric_objects import Triangulation, Point, Triangle
+from scipy.spatial import ConvexHull
+from shapely.geometry import Point as PointShapely
+from shapely.geometry.polygon import Polygon
+from dataclasses import dataclass
+import numpy as np
+
+class Point_list_to_npArray_converter:
+    def list_of_points_to_nparray(self, points: list[Point]):
+            array = np.empty([0, 2])
+            for point in points:
+                array = np.append(array, [np.array([point.x, point.y])], axis=0)
+            return array
```

Comment:
Tohle mi přijde jako duplikace funkce `points_to_numpy_array` z `geometric_objects.py`. Použil spíš ji, pokud to půjde.